tools/bin/ecflow/processes/main/meteorological/tocel_combine.py

```diff
@@ -46,13 +44,15 @@
     np.savetxt(os.path.join(
         final_dir, '{DATA_LAT_LON}'), combined, delimiter='  ', fmt="%.5f")
 
-else:  # if we only downloaded yesterday's data we need to append it to the existing data
+else:  # if we only downloaded the last 60 days we need to append it to the existing data
```

Comment:
The full_year variable is created in the get_metdata.py script, it's set to 'Year' if we need to download the whole year and 'Day' if we can download a shorter period (the 'Day' name comes from when we were downloading only one day, I need to find a new name since now we download either a full year or 60 days).  This is based on whether the directory for the full year data is empty or not. Here, if full_year is not 'Year', it must be 'Day' and thus we append to the existing data.